requirements/tools.txt

```diff
@@ -18,6 +18,7 @@ MarkupSafe==1.0
 mccabe==0.6.1
 numpy==1.12.1
 packaging==16.8
+pip==9.0.1
```

Comment:
Gosh, I assume so. I don't know of any reason why not and it would seem like a pretty bad omission!